spec/debugger_spec.rb

```diff
@@ -64,10 +64,20 @@
   end
 
   describe 'test' do
-    let(:pattern) { '(send _ #eval("puts :x"))' }
+    context 'when the pattern contains an eval' do
+      let(:pattern) { '(send _ #eval("puts :x"))' }
 
-    it 'forbids most function calls' do
-      expect { debugger.test }.to raise_error(/Forbidden function call: eval. Acceptable .* is_a?/)
+      it 'forbids most function calls' do
+        expect { debugger.test }.to raise_error(/Forbidden function call: eval. Acceptable .* is_a?/)
+      end
+    end
+
+    context 'when the pattern contains an end_with?' do
+      let(:pattern) { '(const _ #end_with?("Job"))' }
```

Comment:
Actually, I think I just wanted
```suggestion
      let(:pattern) { '(const _ end_with?("Job"))' }
```
which appears to be permitted.